code-challenges.test.js

```diff
@@ -19,14 +19,42 @@ const numbersArray1 = [6, 7, 8, 9, 10]
 const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
+// describe("multThree", () => {
+//   it("take in an array of numbers and returns an array with all the numbers multipled by 3", () => {
+// expect(multThree(numbersArray1)).toEqual([18, 21, 24, 27, 30])
+//   })
+// })
+//
+// // // pass the variable name into the "function invocation"
+// ex.
+// expect(multThree(numbersArray1)).toEqual("18, 21, 24, 27, 30")
+// expect(multThree(numbersArray2)).toEqual("72, 81, 90, 99, 108")
+//   })
+// })
 
 // b) Create the function that makes the test pass.
 
-
-
-// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
-
-// a) Create a test with expect statements for each of the variables provided.
+// pseudo code:
+// create a function called multThree
+// function multThree will ittrate through an array of numbers using map
+// map through the array
+// // set map argument to be value * 3
+// // return the array of numbers multiplied by three
+//
+// const multThree = (array) => {
+//   return array.map(value => value * 3)
+// }
+
+
+// // 1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
+//
+// // a) Create a test with expect statements for each of the variables provided.
+//
+// describe("divisibleByThree", () => {
+//   it("returns a string saying the number is divisible by three or not", () => {
+// expect(divisibleByThree()).toEqual("The number is or is not divisible by three")
```

Comment:
When making expect statements, you are hard coding exactly what you expect to see and variable should live within the test.

describe("divisibleByThree", () => {
   it("returns a string saying the number is divisible by three or not", () => {
  const num1 = 15
  // Expected output: "15 is divisible by three"
  const num2 = 0
  // Expected output: "0 is divisible by three"
  const num3 = -7
  // Expected output: "-7 is not divisible by three"
expect(divisibleByThree(num1)).toEqual("15 is divisible by three")